pkg/solana/config/multinode.go

```diff
@@ -1,87 +1,206 @@
 package config
 
-import "time"
+import (
+	"time"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/config"
+
+	mn "github.com/smartcontractkit/chainlink-solana/pkg/solana/client/multinode"
+)
+
+// MultiNodeConfig is a wrapper to provide required functions while keeping configs Public
+type MultiNodeConfig struct {
+	MultiNode
+}
 
 type MultiNode struct {
-	// TODO: Determine current config overlap https://smartcontract-it.atlassian.net/browse/BCI-4065
```

Comment:
Did not end up using any shared configs.

Reply:
It would be nice to reuse the config across different chains. The EVM case might be a bit complex, but for new integrations like Aptos/Tron, it does not make sense to duplicate this code.
Could you create a follow up ticket to address that?

Reply:
Yeah sounds good